src/viz/test/CMakeLists.txt

```diff
@@ -52,12 +53,14 @@ if( NOT APPLE )
     DRIVER ${CMAKE_CURRENT_SOURCE_DIR}/tstEnsight_Translator.py
     APP    $<TARGET_FILE_DIR:Ut_viz_tstEnsight_Translator_exe>/$<TARGET_FILE_NAME:Ut_viz_tstEnsight_Translator_exe>
     BUILDENV "CMAKE_HOST_SYSTEM_PROCESSOR=${CMAKE_HOST_SYSTEM_PROCESSOR}"
+    PE_LIST "1;2;4"
```

Comment:
Do we need to add a `RESOURCE_LOCK` to ensure that these tests don't access in/out files simultaneously?

Reply:
All files are unique based on the number of processors.

Reply:
Actually there might be read contention for the serial golds (which happen for every processor count variation) but no write contention. I don't think this should be a real issue (because only a few processor will ever be run at the same time).